examples/statemachine/test/statemachine-cli.test.ts

```diff
@@ -0,0 +1,50 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import path from 'path';
+import fs from 'fs';
+import { exec, ExecException } from 'child_process';
+
+describe('Test the statemachine CLI', () => {
+    let fullPath: string;
+    let fileName = path.join(__dirname, '../example/trafficlight.statemachine');
+    const destination = 'statemachine-example-test';
+
+    test('Generator command returns code 0 and creates expected files', async () => {
+        const result = await cli(['generate', fileName, `-d ${destination}`]);
+        expect(result.code).toBe(0);
+
+        fileName = fileName.replace(/\..*$/, '').replace(/[.-]/g, '');
+        fullPath = path.join(destination, `${path.basename(fileName)}.cpp`);
+        const generatedDirExists = fs.existsSync(fullPath);
+        expect(generatedDirExists).toBe(true);
+    });
+
+    afterAll(() => {
+        fs.rmSync(destination, { recursive: true, force: true });
```

Comment:
This function is not available in Node.js 12.

Reply:
Yo. Just fixed